providers/implementations/asymciphers/rsa_enc.c

```diff
@@ -252,6 +279,14 @@ static int rsa_get_ctx_params(void *vprsactx, OSSL_PARAM *params)
     if (p != NULL && !OSSL_PARAM_set_size_t(p, prsactx->oaep_labellen))
         return 0;
 
+    p = OSSL_PARAM_locate(params, OSSL_ASYM_CIPHER_PARAM_TLS_CLIENT_VERSION);
+    if (p != NULL && !OSSL_PARAM_set_uint(p, prsactx->client_version))
+        return 0;
```

Comment:
Should that be `p == NULL || !OSSL_PARAM_set_uint`

Reply:
No. The `OSSL_ASYM_CIPHER_PARAM_TLS_CLIENT_VERSION` parameter value may not have been requested. If it wasn't requested we skip this. We only fail if it was requested and we fail to set its value.